pkg/login/ext_user.go

```diff
@@ -155,6 +164,18 @@ func updateUser(user *m.User, extUser *m.ExternalUserInfo) error {
 	return bus.Dispatch(updateCmd)
 }
 
+func updateUserAuth(user *m.User, extUser *m.ExternalUserInfo) error {
+	updateCmd := &m.UpdateAuthInfoCommand{
+		AuthModule: extUser.AuthModule,
+		AuthId:     extUser.AuthId,
+		UserId:     user.Id,
+		OAuthToken: extUser.OAuthToken,
+	}
+
+	log.Debug("Updating user_auth info for user_id %d", user.Id)
```

Comment:
we use a none standard log library that allows you to add key value in logs.
So this should something like this
```go 
log.Debug("updating user_auth info", "user_id", user.Id)
```